internal/controller/kubecertagent/deleter_test.go

```diff
@@ -384,16 +308,74 @@ func TestDeleterControllerSync(t *testing.T) {
 						err := controllerlib.TestSync(t, subject, *syncContext)
 
 						r.NoError(err)
-						r.Equal(
-							[]coretesting.Action{
-								coretesting.NewDeleteAction(
-									podsGVR,
-									agentPodNamespace,
-									agentPod.Name,
-								),
-							},
-							kubeAPIClient.Actions(),
-						)
+						requireAgentPodWasDeleted()
+					})
+				})
+
+				when("the agent pod is out of sync with the template via labels", func() {
+					when("an additional label's value was changed", func() {
+						it.Before(func() {
+							updatedAgentPod := agentPod.DeepCopy()
+							updatedAgentPod.ObjectMeta.Labels = map[string]string{
+								agentPodLabelKey: agentPodLabelValue,
```

Comment:
Is this a place where we should follow the testing practice of not using source code constants in a test? Looks liked we use `"kube-cert-agent.pinniped.dev"` in `exampleControllerManagerAndAgentPods()`.

Reply:
Good call, I'll update them all.